packages/puppeteer-core/src/common/Input.ts

```diff
@@ -682,4 +682,42 @@ export class Touchscreen {
       modifiers: this.#keyboard._modifiers,
     });
   }
+
+  /**
+   * Dispatches a `touchstart` event.
+   * @param x - Horizontal position of the tap.
+   * @param y - Vertical position of the tap.
+   */
+  async touchStart(x: number, y: number): Promise<void> {
+    const touchPoints = [{x: Math.round(x), y: Math.round(y)}];
+    await this.#client.send('Input.dispatchTouchEvent', {
+      type: 'touchStart',
+      touchPoints,
+      modifiers: this.#keyboard._modifiers,
+    });
+  }
+  /**
+   * Dispatches a `touchMove` event.
+   * @param x - Horizontal position of the move.
+   * @param y - Vertical position of the move.
+   */
+  async touchMove(x: number, y: number): Promise<void> {
+    const movePoints = [{x: Math.round(x), y: Math.round(y)}];
+    await this.#client.send('Input.dispatchTouchEvent', {
+      type: 'touchMove',
+      touchPoints: movePoints,
+      modifiers: this.#keyboard._modifiers,
+    });
+  }
+  /**
+   * Dispatches a `touchend` event.
+   * stop the touchStart function
```

Comment:
Let's remove this part of the comment and leave only `Dispatches a `touchend` event.`

Reply:
ok